sql/core/src/main/scala/org/apache/spark/sql/execution/window/WindowFunctionFrame.scala

```diff
@@ -77,31 +77,30 @@ object WindowFunctionFrame {
  * @param newMutableProjection function used to create the projection.
  * @param offset by which rows get moved within a partition.
  */
-final class OffsetWindowFunctionFrame(
```

Comment:
can we implement `startWithCurrentRow` logic in the original `OffsetWindowFunctionFrame` with some if-else?

Reply:
We can't uniform the behavior.